packages/nyc-test-coverage/src/index.ts

```diff
@@ -0,0 +1,40 @@
+import { InjectedSinks } from '@temporalio/worker';
+import { CoverageSinks } from './sinks';
+import libCoverage from 'istanbul-lib-coverage';
+
+export class WorkflowCoverage {
```

Comment:
You could add the interceptor modules to this class and maybe even a method to apply to worker options

Reply:
I'm not sure how to do this without having Webpack bundle the entire WorkflowCoverage class. It looks like the only way to pass interceptors is as a workflow module: https://github.com/temporalio/sdk-typescript/blob/0310ad6e44ea49220e3ac14fa90c3e8c78ec3647/packages/worker/src/interceptors.ts#L42-L55 , so I would need a file path for the interceptor modules. Am I missing something?